tests/unit/ceed/test_ceed.cpp

```diff
@@ -20,11 +20,25 @@ using namespace mfem;
 namespace ceed_test
 {
 
+#ifdef MFEM_USE_CEED
+
 double coeff_function(const Vector &x)
 {
    return 1.0 + x[0]*x[0];
 }
 
+// Velocity coefficient
+void velocity_function(const Vector &x, Vector &v)
+{
+   int dim = x.Size();
+   switch (dim)
+   {
+      case 1: v(0) = 1.0; break;
+      case 2: v(0) = 1.0; v(1) = 1.0; break;
+      case 3: v(0) = 1.0; v(1) = 1.0; v(2) = 1.0; break;
+   }
```

Comment:
I think this is why the tests didn't find the bug, this was too simple of a vector coefficient.

Reply:
Aha, yeah, I think it would be good to double check that a variable coefficient would have really caught the problem so we can be confident that our unit tests are robust.

Reply:
Did you have the chance to see if the unit tests with the new velocity would have caught the bugs from before?